<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>билет 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>билет 2</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>261</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>чет</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>нечет</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2_1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти сумму</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Результат:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_res_2_1">
      <property name="geometry">
       <rect>
        <x>336</x>
        <y>10</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>билет 3</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_3_1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>sum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mult</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBox_3_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>k:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3_1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>354</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Результат:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_res_3_1">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>билет 4</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>билет 5</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>билет 6</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>билет 7</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
